samples/wifi/scan/Kconfig

```diff
@@ -16,6 +16,12 @@ config WIFI_MAC_ADDRESS
 		use the default MAC address (locally administrative). This is only used
 		when the nRF7002 OTP is not programmed with a MAC address.
 
+config WIFI_SCAN_INTERVAL
```

Comment:
you could embed the time unit in the Kconfig name, but that's nit picking

Reply:
No, its a good point, will do, thanks.